tests/spec/enable_http2_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature "Enabling HTTPS" do
```

Comment:
This probably doesn't belong in this PR.

Reply:
Not sure how that happened. I removed it now. Thanks for pointing it out.